object_visualizer/src/object_visualizer_node.cpp

```diff
@@ -73,6 +84,11 @@ namespace object_visualizer
     get_parameter<std::string>("roadway_obstacles_viz_ns", config_.roadway_obstacles_viz_ns);
     get_parameter<uint8_t>("marker_shape", config_.marker_shape);
 
+    // Load new parameters for pedestrian visualization
```

Comment:
I think this comment was just included as part of initial prototyping, can we remove and group all `get_parameter` lines?